### PR TITLE
Remove the fragment

### PR DESCRIPTION
**Pull request by kevva**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -24,6 +24,7 @@ module.exports = function (str) {
 	// prevent these from being used by `url.format`
 	delete urlObj.host;
 	delete urlObj.query;
+	delete urlObj.hash;
```
Comment (sindresorhus) [inline index.js]: Can you add a comment as this is deleted for a different reason?

Comment (kevva) [inline index.js]: Hehe, I did, but then I changed :PpP. Knew it'd fuck things up.

**Comment by sindresorhus** [comment]
Can you resolve the merge conflict?

**Comment by kevva** [comment]
Yup, done.

**Pull request merged**